README.md

```diff
@@ -0,0 +1,14 @@
+## Hello World Project Documentation
+
+    this project consist to set-up linters in github and create basic HTML page and style it!
+    
+## Built With
+
+- HTML and CSS
+-VSCode
+
+
+## Authors
+
+- GitHub: [@nmeziany][https://github.com/nmeziany/PortfolioSkeleton]
+- linkedIn: [https://www.linkedin.com/in/noureddine-meziany/]
```

Comment:
- [ ] [REQUIRED] You have done well implementing the README file however, it is not that descriptive yet. You left out the project description, MIT license, and the project image. Please see [Microverse readme-template here](https://github.com/microverseinc/readme-template) for more explanation.

Reply:
Modifications done for README file